packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -24,9 +24,9 @@
   "baseFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
   "l1FeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
   "sequencerFeeVaultMinimumWithdrawalAmount": "0x8ac7230489e80000",
-  "baseFeeVaultWithdrawalNetwork": "remote",
-  "l1FeeVaultWithdrawalNetwork": "remote",
-  "sequencerFeeVaultWithdrawalNetwork": "remote",
+  "baseFeeVaultWithdrawalNetwork": 0,
+  "l1FeeVaultWithdrawalNetwork": 0,
+  "sequencerFeeVaultWithdrawalNetwork": 0,
```

Comment:
These used to be `"remote"`, but judging by the existing Go code and other deploy configs, I think it's supposed to be `0`

Reply:
the go code will parse the string into a solidity compatible type

Reply:
`make devnet-allocs` executes `DeployConfig.s.sol` and it expects these properties to be `uint256`. Unless it's an issue, I think it's simpler if these properties are `0`. It was also the only deploy config where they were set to `remote` or `local`
<img width="481" alt="image" src="https://github.com/ethereum-optimism/optimism/assets/20375610/f75ddc9c-374d-414c-8565-a0fc8ebbfe61">